.github/workflows/ci.yml

```
@@ -239,10 +245,59 @@ jobs:
       - name: Display MAPDL Logs
         if: always()
         run: cat log.txt
+  
+  windows_build_test:
+    name: Windows Build and Unit Testing
+    needs: testimport
+    runs-on: Windows
+    strategy:
+      matrix:
+        mapdl-version: ['v21.2.1']
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Create virtual env
+        run: |
+          python -m venv testenv
+          testenv\Scripts\Activate.ps1
+          python -m pip install pip -U
+          python -m pip install wheel setuptools -U
+          python -c "import sys; print(sys.executable)"
+
+      - name: Install ansys-mapdl-core
+        run: |
+          testenv\Scripts\Activate.ps1
+          pip install -r requirements_build.txt
+          pip install . --use-feature=in-tree-build
+          cd tests  # run from test directory to test installed module
+          python -c "from ansys.mapdl import core as pymapdl; print(pymapdl.Report())"
+
+      - name: Unit Testing
+        run: |
+          testenv\Scripts\Activate.ps1
+          pip install -r requirements_test.txt
+          pip install pytest-azurepipelines
+          pytest -v --junitxml=junit/test-results.xml --cov --cov-report=xml --cov-report=html
+        env:
+          ANSYSLI_SERVERS: ${{ secrets.WINDOWS_LICENSE_SERVER }}
```

Should be able to just use:

```suggestion
          ANSYSLI_SERVERS: ${{ secrets.LICENSE_SERVER }}
```